modules/gapi/src/executor/gstreamingexecutor.hpp

```diff
@@ -79,6 +79,11 @@ class GStreamingExecutor final
 
     cv::gimpl::GIslandModel::Graph m_gim; // FIXME: make const?
 
+    cv::GCompileArgs comp_args;
+    bool wasFinished = false;
```

Comment:
Please follow the same `m_` naming convention for member class variables as everywhere.

Reply:
Fixed